src/cartesi-machine.lua

```diff
@@ -1999,60 +1936,62 @@ local function ilog2(value)
     return value
 end
 
-local function check_rollup_memory_range_config(range, name)
-    assert(range, string.format("rollup range %s must be defined", name))
-    assert(not range.shared, string.format("rollup range %s cannot be shared", name))
+local function check_cmio_memory_range_config(range, name)
+    assert(range, string.format("cmio range %s must be defined", name))
+    assert(not range.shared, string.format("cmio range %s cannot be shared", name))
     assert(
         is_power_of_two(range.length),
-        string.format("rollup range %s length not a power of two (%u)", name, range.length)
+        string.format("cmio range %s length not a power of two (%u)", name, range.length)
     )
     local log = ilog2(range.length)
     local aligned_start = (range.start >> log) << log
     assert(
         aligned_start == range.start,
-        string.format("rollup range %s start not aligned to its power of two size", name)
+        string.format("cmio range %s start not aligned to its power of two size", name)
```

Comment:
Minor thing here.
This assert is not doing anything, I think.
The current implementation of ilog2() always returns 0.

Reply:
```
local function ilog2(value)
    value = assert(math.tointeger(value), "expected integer")
    assert(value ~= 0, "expected non-zero integer")
    local log = 0
    while value > 1 do
        log = log + 1
        value = value >> 1
    end
    return log
end
```

Reply:
I'll push a commit there with this fix.

Reply:
Done.